package.json

```diff
@@ -10,7 +10,8 @@
     "build:starters": "pnpm --filter ./starters run build",
     "build:all": "pnpm recursive run build",
     "generate-docs": "pnpm --filter ./web clear && pnpm --filter ./web build",
-    "postinstall": "pnpm build:pkgs"
+    "postinstall": "pnpm build:pkgs",
```

Comment:
@CobyPear yeah, the docs for this seem unnecessarily confusing. If I'm parsing things correctly, the workflow is going to run the scripts for:

```
pnpm ci:version
pnpm ci:publish
```

and the scripts should be:

```
"ci:version": "changeset version",
"ci:publish": "changeset publish",
```

I technically haven't run `changeset publish` before. I used `pnpm publish -r`, but the docs seem to imply that both should work.